packages/storefront-lwc/src/modules/commerce/checkout/checkout.js

```diff
@@ -0,0 +1,58 @@
+import { LightningElement, wire, track } from 'lwc';
+import { routeParams, history } from '@lwce/router';
+
+export default class Checkout extends LightningElement {
+    @wire(routeParams) routeParams;
+    @wire(history) history;
+
```

Comment:
I think we should also get the basket and that info will be passed to order totals component and product summary components.